src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -2021,7 +2021,13 @@ void Interpolate::initSupportedPrimitiveDescriptors() {
         return;
 
     ov::element::Type inputPrecision = getOriginalInputPrecisionAtPort(DATA_ID);
-    if ((inputPrecision != ov::element::i8) && (inputPrecision != ov::element::u8) && (inputPrecision != ov::element::bf16)) {
+
+#if defined(OV_CPU_WITH_ACL)
+    bool isInputPrecisionSupported = one_of(inputPrecision, ov::element::i8, ov::element::u8, ov::element::bf16, ov::element::f16);
```

Comment:
I don't see any limitations on supported precision inside ACL Interpolate Executor (btw why?), but I believe bf16 shouldn't be in the list. Maybe smt else.
BTW does Intepolate test on ARM covers all these precisions?

Reply:
Right, bf16 is removed, i16 is added.
List of supported ACL data type configuration here: https://github.com/ARM-software/ComputeLibrary/blob/main/arm_compute/runtime/NEON/functions/NEScale.h#L60-L69

Reply:
SLT interpolate tests are not enabled on ARM.